iOS/Multiplatform Redux Sample/Multiplatform Redux Sample/Map/PinChangeSet.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  PinChangeSet.swift
+//  Multiplatform Redux Sample
+//
+//  Created by Samuel Bichsel on 05.03.21.
+//
+
+import Foundation
+import MapKit
+import ReduxSampleShared
+
+struct PinChangeSet {
+    let mapView: MKMapView
+    let newPoints: Set<CollectionPoint>
+    let existing: Set<CollectionPoint>
+
+    init(mapView: MKMapView, collectionPoints: [CollectionPoint]) {
+        self.mapView = mapView
+        self.newPoints = Set(collectionPoints)
+        existing = Set(mapView.annotations.compactMap { $0 as? CollectionPoint })
+    }
+
+    private var add: Set<CollectionPoint> { mutating get { newPoints.subtracting(existing) } }
+    private var remove: Set<CollectionPoint> { mutating get { existing.subtracting(newPoints) } }
+
+    mutating func updateAnnotations(selection: CollectionPoint?) {
```

Comment:
You don't need to mark this function as mutating if you make the getters nonmutating.